src/Utils.test.js

```diff
@@ -37,3 +42,30 @@ test('Test extend', () => {
     expect(extend({ a: 0 }, {b: 1}, { c: 2})).toEqual({ a: 0, b: 1, c: 2});
     expect(extend({ foo: { bar: { baz: true }}}, { foo: { new: true, bar: { new: true, baz: false}}})).toEqual({ foo: { new: true, bar: { new: true, baz: false}}});
 });
+
+test('Test focusByQuerySelector', () => {
+    // Test fixture with three focusable elements.
+    const testFixture = mount(
+        <div>
+            <a className="first" href="http://create.weavly.org/">First</a>
+            <a className="second" href="http://create.weavly.org/">Second</a>
+            <a className="third" href="http://create.weavly.org/">Third</a>
+        </div>,
+        // $FlowFixMe:  Flow is worried that document.body might not exist.
+        {attachTo: document.body}
+    );
+
+    const secondElement = testFixture.find(".second");
+
+    // We should be focused on the body by default.
+    // $FlowFixMe: Flow is worried that document.activeElement might not exist.
+    expect(document.activeElement.localName).toEqual('body');
```

Comment:
Do we need to test this? As you commented, I think focus on body is default expected behaviour.

Reply:
I'm just making sure it's not focused on the element before I change focus.

Reply:
It's a little superstitious, I know.